cmd/tailscale/cli/web.go

```diff
@@ -83,6 +85,35 @@ var webArgs struct {
 	cgi    bool
 }
 
+func tlsCertFromEnvironment(listen string) (*http.Server, error) {
+	crt := os.Getenv("TLS_CRT")
+	key := os.Getenv("TLS_KEY")
+	if crt != "" && key != "" {
```

Comment:
Go in general prefers early returns and outdenting

so inverting this,

```go
   if crt == "" || key == "" {
       return nil, nil
   }
   ....
```

Reply:
Done.